include/unifex/async_scope.hpp

```diff
@@ -170,6 +170,14 @@ struct async_scope {
     }
   }
 
+  template (typename Fun, typename Scheduler)
+    (requires callable<Fun> && scheduler<Scheduler>)
+  void spawn_with(Fun&& fun, Scheduler&& scheduler) {
+    spawn(
+      transform(schedule(), (Fun&&) fun),
+      (Scheduler&&) scheduler);
```

Comment:
Since you're invoking `spawn(sender, scheduler)` and not `spawn(sender)`, isn't it the case that you should either use `just()` instead of `schedule()` here, or use `schedule(scheduler)` but invoke the unary `spawn`?
```suggestion
    spawn(
      transform(just(), (Fun&&) fun),
      (Scheduler&&) scheduler);
```
```suggestion
    spawn(
      transform(schedule((Scheduler&&) scheduler), (Fun&&) fun));
```